css/style.css

```diff
@@ -38,11 +40,17 @@ Developer @GA in NYC
   padding: 0 10px 10px 0; /* note the padding around a left floated image */
 }
 
+.firstcharacter { float: left; font-size: 75px; line-height: 60px; padding-top: 4px; padding-right: 8px; padding-left: 3px;
```

Comment:
see if you can do this without the `class`. 

see this site: http://www.w3schools.com/cssref/sel_firstletter.asp